mirage-types,io-page,mirage-types-lwt cyclic dependencies

```
$ opam update
[...]
[WARNING] A conflict was detected in your installation. This can be caused by updated
constraints in your installed packages:

You should run "opam upgrade --fixup" to resolve the situation.
$ opam upgrade --fixup
$ echo $?
0
$ opam upgrade
[ERROR] Dependency errors in the upgrade actions. Please update, and report the
following to the package maintainers if the error persists:
  - recompile mirage-types.2.0.1 -> upgrade   io-page from 1.1.1 to 1.2.0 -> recompile mirage-types-lwt.2.0.1 -> recompile mirage-types.2.0.1
  - upgrade   io-page from 1.1.1 to 1.2.0 -> recompile mirage-types-lwt.2.0.1 -> upgrade   io-page from 1.1.1 to 1.2.0
You may try upgrading packages individually to work around this.

$ opam config report
# OPAM status report
# opam-version    1.2.0
# self-upgrade    no
# os              linux
# external-solver aspcud
# criteria        -count(removed),-notuptodate(request),-count(down),-notuptodate(changed),-count(changed),-notuptodate(solution)*
# jobs            9
# repositories    1* (http), 1 (local), 1 (version-controlled)
# pinned          1 (version control)
# current-switch  4.02.1
# last-update     2014-11-30 10:31

$ opam install io-page.1.2.0
The actions to process have cyclic dependencies:
  - upgrade   io-page from 1.1.1 to 1.2.0 -> recompile mirage-types.2.0.1 -> recompile mirage-types-lwt.2.0.1 -> upgrade   io-page from 1.1.1 to 1.2.0

No solution found, exiting
```

--- a/packages/io-page/io-page.1.2.0/opam
+++ b/packages/io-page/io-page.1.2.0/opam
@@ -20,6 +20,5 @@ depends: [
   "ocamlfind"
   "cstruct" {>="1.0.1"}
   "ounit" {test}
-  "mirage-types-lwt" {test}
 ]
 available: [ocaml-version >= "4.00.0"]

